src/templates/view-edit-toolbar.html

```
@@ -171,12 +171,12 @@ <h5 class='modal-title' id='procurementRequestModalLabel'>{{ 'Create procurement
 
   {% if mode == 'edit' %}
     {# VIEW button #}
-    <a title='{{ 'View mode'|trans }}' aria-label='{{ 'View mode'|trans }}' href='?mode=view&amp;id={{ Entity.id }}' class='btn btn-primary hl-hover-gray p-2 mr-2  lh-normal border-0'>
+    <button type='button' title='{{ 'View mode'|trans }}' aria-label='{{ 'View mode'|trans }}' href='?mode=view&amp;id={{ Entity.id }}' class='btn btn-primary p-2 mr-2 lh-normal'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)